common/vueapp/UserService.js

```diff
@@ -159,12 +159,14 @@ export default {
         reverse = true;
         r = r.substring(1);
       }
-      if ((!reverse && !user.roles.includes(r)) ||
```

Comment:
please write a test for this that fails with the old code and works with this new code. this hurts my head.